workflows/tests/aligner/bwa.nf.test

```diff
@@ -0,0 +1,97 @@
+nextflow_pipeline {
+
+    name "BWA"
+    script "../../../main.nf"
+    tag "aligner"
+    tag "bwa"
+
+    test("Should run with defaults") {
+
+        when {
+            params {
+                outdir = "$outputDir"
+                aligner = "bwa"
+                input = "${projectDir}/assets/samplesheet.csv"
+            }
+        }
+
+        then {
+            assertAll(
+                { assert workflow.success },
+                { assert snapshot(UTILS.removeNextflowVersion("$outputDir/pipeline_info/nf_core_pipeline_software_mqc_versions.yml")).match("software_versions") },
+                { assert snapshot(
+                    workflow.trace.tasks().size(),
+                    path("$outputDir/bbmap").list(),
+                    // BUG https://github.com/nf-core/nascent/issues/102
```

Comment:
I guess this is intentionally left with comments, isn't it?

Reply:
Yeah, I hadn't figured out the test flaking out. I also have a PR coming up to refactor the outputs as well, so it's a bit of a waste to try to get the tests working when it's all getting reworked.